### ISSUE TITLE
Distorted voice.

### ISSUE THREAD

**Issue by Sangweenie** [Wurielle/izabela-desktop#18]
My friend has been using this program for a while now and it is very helpful. However, recently they have been having an issue where the voice sounds very different on their end compared to the people hearing it over discord/other VoIP programs. It seems to have somewhat fixed itself, now sounding a lot less distorted but the TTS still speaks in a deeper tone compared to client-side which wasn't a problem before.  Is there a way to fix this or is it normal with the program?

**Comment by Wurielle** [comment]
Hello @Sangweenie! Thanks for letting me know about the issue!

Is it possible  for you to provide me with the exact settings used when this issue happened?
Screenshots are fine, I'd like to be able to reproduce it if possible. Thanks in advance!

**Comment by Wurielle** [comment]
Hello again @Sangweenie! Sorry for the delay this time!

Thanks a lot for the audio samples! I noticed the audio output used by your friend is the "Hi-Fi Cable Input (VB-Audio Hi-Fi Cable)". The one recommended for the app is the regular VB-Cable that can be found [on the official website](https://www.vb-audio.com/Cable/).

I'm not sure if it is what's causing the issue but I looked around the VB-Audio Hi-Fi Cable section and it seems like it needs some sort of configuration out of the box which I'm not too familiar with.

I recommend trying again with the regular VB-Cable to see if it sounds more as you'd expect. I'll try to install the Hi-Fi cable and see what's the required configuration to make it run well with the app in the meantime.

Let me know if it helped!
